org.eclipse.lsp4j.jsonrpc/src/main/java/org/eclipse/lsp4j/jsonrpc/services/GenericEndpoint.java

```diff
@@ -62,6 +65,17 @@ protected void recursiveFindRpcMethods(Object current, Set<Class<?>> visited, Se
 			}
 		});
 	}
+	
+	protected Object[] getArguments(Method method, Object arg) {
+		if (arg == null) {
+			return NO_ARGUMENTS;
+		}
+		if (method.getParameterCount() == 0) {
+			LOG.warning("Unexpected params '" + arg + "' for '" + method + "' is ignored");
```

Comment:
actually the value does not matter, the point that the argument should not passed at all